lib/xls_function/evaluators/functions/switch.rb

```diff
@@ -0,0 +1,29 @@
+module XlsFunction
+  module Evaluators
+    module Functions
+      class Switch < ::XlsFunction::Evaluators::FunctionEvaluator
+        function_as :switch
+
+        def eval_arglist
+          # Skip common argument evaluation and assignment for short-circuit
+        end
+
+        def eval
+          condition = arg_list.first.evaluate(context)
+          count = 0
+
+          arg_list[1..-1].each_slice(2) do |expr, value|
+            count += 1
+            break if count > 126
+
+            return value&.evaluate(context) if expr&.evaluate(context) == condition
+
+            return expr&.evaluate(context) if value&.evaluate(context).nil?
+          end
+
+          XlsFunction::ErrorValue.na
```

Comment:
I'd like to add some messages with class_info. See also other ErrorValue usage.

Reply:
I added a message in the commit below.
[a230f0f](https://github.com/moneyforward/xls_function/pull/16/commits/a230f0f56f69509614d2d7992a58e556cd9f2da7)